app/static/css/main.css

```diff
@@ -123,7 +123,7 @@ textarea {
     }
 }
 
-.scrollToTopBtn {
+.scrollTopBtn {
```

Comment:
Can you rename this to `scroll_top_button` (elsewhere we use snake_case for IDs and classes) and move it to `controls.css`, since it's only relevant to the homepage/places where we have the searching controls? Also put it before any `@media` queries as those are at the end to distinguish the mobile styles from the normal styles.